scripts/publish/installers/install.sh

```diff
@@ -21,6 +21,8 @@ function error() {
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; then
```

Comment:
```suggestion
if [[ "$OSTYPE" == "linux"* ]]; then
```
Why not just to adjust the original if ?